### DISCUSSION TITLE
Nothing fancy, looking for ideas

### DISCUSSION THREAD

**Discussion by Gregalot** [wlcrs/huawei_solar#238] [Show your dashboards!]
![image](https://user-images.githubusercontent.com/117384097/210766567-c7129c77-6810-492b-ad42-7f36b872e878.png)

**Comment by Rhysgh** [comment]
![image](https://user-images.githubusercontent.com/57257743/210771915-36c1168b-a4c6-4cb4-92e1-dbda63568bd3.png)
